src/syrupy/filters.py

```diff
@@ -13,7 +13,7 @@ def paths(path_string: str, *path_strings: str) -> "PropertyFilter":
     Factory to create a filter using list of paths
     """
 
-    def path_filter(prop: "PropertyName", path: "PropertyPath") -> bool:
+    def path_filter(*, prop: "PropertyName", path: "PropertyPath") -> bool:
```

Comment:
(note) should capture this in a "migration" guide (which can also just be a longer comment in the changelog)

Reply:
I could add it, but I didn't think there was a need as the function is already always called with keyword arguments. Also this function is returned by the factory, not really useful except when passed to the assertion